### DISCUSSION TITLE
Time scale of the inferred rate matrix Q

### DISCUSSION THREAD

**Discussion by perryXuu** [iqtree/iqtree2#202] [Q&A] [answered]
Hi everyone,

I'm trying to understand the substitution rate matrix inferred (in the .iqtree file).

Rate matrix Q:

  A    -1.248    0.3763    0.6966    0.1752
  C    0.1777   -0.8777    0.3713    0.3287
  G    0.3334    0.3763   -0.8849    0.1752
  T    0.1777    0.7059    0.3713    -1.255
 The date I provided is in YYYY-MM-DD format, and I wonder what is the time scale for this rate matrix. Is it in Year automatically? Didn't find this information in the documentation, I would be really appreciating any help!! Thank you!!

Best,
Perry

**Comment by roblanf** [comment, marked as answer]
The rate matrix does not have units or a timescale. These are relative values, also known as instantaneous rates.

You can convert these into probabilities of change along a branch of length `t` by exponentiating this matrix. The RevBayes documentation is quite good on this (and there are videos too, though I haven't watched them!): 

https://revbayes.github.io/tutorials/ctmc/

As a side note (but an important one in this context) please note that in IQ-TREE, molecular dating is done by a separate program, LSD2, which is run *after* IQ-TREE finishes. LSD2 relies on the branch lengths and time constraints you supply. Thus, IQ-TREE itself does not do anything with your date information when calculating the Q matrix. Indeed, IQ-TREE is completely ignorant of any notion of absolute time.

**Reply by perryXuu** [reply]
Thank you so much! That was very helpful!!
